### ISSUE TITLE
Customized file upload buttons are not accessible with keyboard navigation

### ISSUE THREAD

**Issue by elysiagabe** [Lambda-School-Labs/My-School-FE#112]
Thanks to Chakra UI, the majority of components are accessible with keyboard navigation and screen readers. However, we had to customize the file upload buttons because Chakra UI does not currently have a file upload component. The way it is customized is not accessible via keyboard navigation. Some research may need to go into this to fix.
